python/pyspark/sql/connect/client.py

```diff
@@ -21,18 +21,20 @@
 import typing
 import uuid
 
-import grpc
+import grpc  # type: ignore
 import pandas
 import pandas as pd
 import pyarrow as pa
 
 import pyspark.sql.connect.proto as pb2
 import pyspark.sql.connect.proto.base_pb2_grpc as grpc_lib
+import pyspark.sql.types
```

Comment:
This import will be against the principle of Connect clients should be a light module without requiring install pyspark?

Reply:
Though I saw `from pyspark import cloudpickle` below which I didn't notice in the past. 

Maybe at this moment, this python client already cannot fully remove the dependency on pyspark?

Reply:
Right now we don't ship this independently. We need to iron out the packaging for Python over time.

Reply:
Sounds good.